modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -5,20 +5,66 @@ This module is included in the following assemblies:
 
 :_mod-docs-content-type: PROCEDURE
 [id="ossm-configuring-distr-tracing-tempo_{context}"]
-= Configuring the distributed tracing platform (Tempo)
+= Configuring the distributed tracing platform (Tempo) and the Red Hat build of OpenTelemetry
 
-You can expose tracing data to the {TempoName} stack by appending a named element and the `zipkin` provider to the `spec.meshConfig.extensionProviders` specification in the `ServiceMehControlPlane`, as shown in the following example. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
+You can expose tracing data to the {TempoName} by appending a named element and the `opentelemetry` provider to the `spec.meshConfig.extensionProviders` specification in the `ServiceMehControlPlane`. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the OpenTelemetry Collector endpoint.
 
-You can create a TempoStack instance in the `tracing-system` namespace _after_ creating the `ServiceMeshControlPlane` and the `ServiceMeshMemberRoll` resources.
+//There is no common attribute for OpenTelemetry Collector.
+
+You can create a {OTELName} instance in a mesh namespace and configure it to send tracing data to a tracing platform backend service.
 
 .Prerequisites
 
-* You have installed the {TempoOperator} and {SMProductName} Operator in the `openshift-operators` namespace.
-* You have created namespaces such as `istio-system` and `tracing-system`.
+* You have created a TempoStack instance using the Red Hat {TempoOperator} in the `tracing-system` namespace. For more information, see "Installing the distributed tracing platform (Tempo)" in the "Additional resources" section.
+
+* You have installed the {OTELOperator} in the namespace `openshift-operators`. For more information, see "Installing the {OTELName}" in the "Additional resources" section.
```

Comment:
is it required to have the OTEL Operator in `openshift-operators` namespace? Because, by default, it is recommended by OTEL bundle to be installed into `openshift-opentelemetry-operator` namespace
![image](https://github.com/openshift/openshift-docs/assets/16251792/a9307913-4147-4bb2-a6e1-7e987869333f)

Reply:
@yxun can you confirm if `openshift-operator` needs to be updated to `openshift-opentelemetry-operator`? 

`openshift-opentelemetry-operator` is also used in Distributed Tracing's instructions: https://docs.openshift.com/container-platform/4.15/observability/otel/otel-installing.html 

Might have been an update since you wrote out your instructions earlier in the month.

Reply:
OTEL operator can be installed in either namespace. Because we install our operators (OSSM operator and Kiali operator) in `openshift-operators` namespace, it will be easier for managing them in the same namespace.

We can mention in our doc that users can install OTEL operator in either the default namespace or the `operator-operators` namespace.